listview/tsfmt.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
This file should not be committed

Reply:
This is an autogenerated file that sets a "common" style when editing the demo-angular via VS Code. I think we should leave those files committed to enforce the same code style and formatting.